src/slack.js

```diff
@@ -48,14 +48,22 @@ const updateSlackStatus = async ({ token, text, emoji }) => {
  * or not the other promises have resolved.
  * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all
  */
-module.exports = (isInMeeting = false, workspaces = slackWorkspaces) =>
+module.exports = (eventUserMail = '', isInMeeting = false, workspaces = slackWorkspaces) =>
   axios.all(
     workspaces.map((slackWorkspace) => {
-      const status = isInMeeting ? 'meetingStatus' : 'noMeetingStatus'
-      return updateSlackStatus({
-        token: slackWorkspace.token,
-        text: slackWorkspace[status].text,
-        emoji: slackWorkspace[status].emoji,
-      })
-    }),
-  )
+      logger('userEmail', slackWorkspace.userMail)
+      logger('eventUserMail', eventUserMail)
```

Comment:
```suggestion
```

Also in L57 and L65 below, please.

Let's not create too much "noise" in the app logs right now, ok?

I've to think about implementing https://www.npmjs.com/package/debug (or something similar) enabling log-levels for debugging, warn and errors.

Reply:
Fine for me, I will remove the log statements. But I added one to indicate if the slack workspace is filtered because of the mail address. Otherwise, the log is misleading.